src/mygrad/__init__.py

```diff
@@ -29,6 +29,8 @@
     cumprod,
     cumsum,
     mean,
+    _canonicalize_weights,
+    average,
```

Comment:
`average` is not a method of `numpy.ndarray` so we ought not include it as a method on `Tensor`

```suggestion
```